src/sdk/sql_cluster_router.cc

```diff
@@ -4068,7 +4067,7 @@ static const std::initializer_list<std::string> GetTableStatusSchema() {
     static const std::initializer_list<std::string> schema = {
         "Table_id",         "Table_name",     "Database_name",     "Storage_type",      "Rows",
         "Memory_data_size", "Disk_data_size", "Partition",         "Partition_unalive", "Replica",
-        "Offline_path",     "Offline_format", "Offline_deep_copy", "Warnings"};
+        "Offline_path",     "Offline_format", "Offline_symbolic_paths", "Warnings"};
```

Comment:
delete `deep_copy` ? SQL semantic breaks.

Reply:
deep copy is an option in load data, not an needed field in table info. Actually, in proto, deep_copy is deprecated.